### PR TITLE
Release v1.25.0

### PR DESCRIPTION
**Pull request by tanmay-db**
## Changes

 * Added `IS_OWNER` permission for SQL Warehouse ([#2600](https://github.com/databricks/terraform-provider-databricks/pull/2600)).
 * Added `managed_identity_id` to [databricks_storage_credential](https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/storage_credential) to support user-assigned managed identities ([#2536](https://github.com/databricks/terraform-provider-databricks/pull/2536)).
 * Added `options` field to UC catalog resource to support foreign catalog creation ([#2616](https://github.com/databricks/terraform-provider-databricks/pull/2616)).
 * Detected run_as drift in job resource ([#2626](https://github.com/databricks/terraform-provider-databricks/pull/2626)).
 * Removed callback field from acceptance test framework ([#2649](https://github.com/databricks/terraform-provider-databricks/pull/2649)).
 * Fixed `force_new` attributes for [databricks_schema](https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/schema) and [databricks_volume](https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/volume) ([#2635](https://github.com/databricks/terraform-provider-databricks/pull/2635)).
 * Fixed do not attempt to delete default schema for foriegn catalogs ([#2622](https://github.com/databricks/terraform-provider-databricks/pull/2622)).

 Documentation:
 * Updated documentation for marketplace admin role ([#2638](https://github.com/databricks/terraform-provider-databricks/pull/2638)).
 * Updated documentation for share and share recipient ([#2641](https://github.com/databricks/terraform-provider-databricks/pull/2641)).
 * Updated documentation for group, service principal and user([#2644](https://github.com/databricks/terraform-provider-databricks/pull/2644)).

 Other Changes:
 * Bumped github.com/databricks/databricks-sdk-go from 0.17.0 to 0.19.1 ([#2660](https://github.com/databricks/terraform-provider-databricks/pull/2660)).
 * Bumped github.com/hashicorp/hcl/v2 from 2.17.0 to 2.18.0 ([#2636](https://github.com/databricks/terraform-provider-databricks/pull/2636)).
 * Added doc strings for ResourceFixtures ([#2633](https://github.com/databricks/terraform-provider-databricks/pull/2633)).

## Tests

All integration tests passed except `AccountServicePrincipalRuleSetsFullLifeCycle` which seems to be failing due to unrelated reasons. This test passed 5 hours ago and there has been no change in master branch since 3 days ago so most likely flaky.

- [ ] `make test` run locally
- [ ] relevant change in `docs/` folder
- [ ] covered with integration tests in `internal/acceptance`
- [ ] relevant acceptance tests are passing
- [ ] using Go SDK

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+ * Add doc strings for ResourceFixtures 
+([#2633](https://github.com/databricks/terraform-provider-databricks/pull/2633)).
+ * Bump github.com/hashicorp/hcl/v2 from 2.17.0 to 2.18.0 
+([#2636](https://github.com/databricks/terraform-provider-databricks/pull/2636)).
+ * Detect run_as drift in job resource 
+([#2626](https://github.com/databricks/terraform-provider-databricks/pull/2626)).
+ * Refactor 
+[databricks_recipient](https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/recipient) 
+to Go SDK 
+([#2592](https://github.com/databricks/terraform-provider-databricks/pull/2592)).
```
Comment (nkvuong) [inline CHANGELOG.md:16]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+ * Remove callback field from acceptance test framework 
+([#2649](https://github.com/databricks/terraform-provider-databricks/pull/2649)).
+ * Revert "Refactor 
+[databricks_recipient](https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/recipient) 
+to Go SDK 
+([#2592](https://github.com/databricks/terraform-provider-databricks/pull/2592))" 
+([#2655](https://github.com/databricks/terraform-provider-databricks/pull/2655)).
```
Comment (nkvuong) [inline CHANGELOG.md:23]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,35 @@
...
+ * Update docs for marketplace admin role 
+([#2638](https://github.com/databricks/terraform-provider-databricks/pull/2638)).
+ * fix: do not attempt to delete default schema for foriegn catalogs 
+([#2622](https://github.com/databricks/terraform-provider-databricks/pull/2622)).
+ * terrafmt; updated share and share recipient docs 
+([#2641](https://github.com/databricks/terraform-provider-databricks/pull/2641)).
+ * update documentation 
+([#2644](https://github.com/databricks/terraform-provider-databricks/pull/2644)).
```
Comment (nkvuong) [inline CHANGELOG.md:31]:
```suggestion
 * various documentation updates ([#2644](https://github.com/databricks/terraform-provider-databricks/pull/2644), [#2641](https://github.com/databricks/terraform-provider-databricks/pull/2641), [#2638](https://github.com/databricks/terraform-provider-databricks/pull/2638), [#2633](https://github.com/databricks/terraform-provider-databricks/pull/2633)).
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,35 @@
...
+([#2592](https://github.com/databricks/terraform-provider-databricks/pull/2592)).
+ * Remove callback field from acceptance test framework 
+([#2649](https://github.com/databricks/terraform-provider-databricks/pull/2649)).
+ * Revert "Refactor 
+[databricks_recipient](https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/recipient) 
+to Go SDK 
+([#2592](https://github.com/databricks/terraform-provider-databricks/pull/2592))" 
+([#2655](https://github.com/databricks/terraform-provider-databricks/pull/2655)).
+ * Update docs for marketplace admin role 
+([#2638](https://github.com/databricks/terraform-provider-databricks/pull/2638)).
```
Comment (nkvuong) [inline CHANGELOG.md:25]:
```suggestion
```

**Review by nkvuong — COMMENTED** [review]
couple of suggestions to changelog

**Comment by tanmay-db** [comment]
@nkvuong Thanks for the review, yeah I haven't updated the changelog yet (for example add -> added, spaces, separation into Documentation, other changes etc...) because we are not doing a release on Friday as a policy and will do on Monday.

There might be some PRs that get merge so will have to rebase and run it again which will lose changes. I will update it on Monday and send for review.

**Review by nkvuong — APPROVED** [review]
lgtm

**Review by nfx — APPROVED** [review]

**Pull request merged**